src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ CHANGELOG
 -----
 
  * added support for hyperlinks
+ * added an `iterate` method to `ProgressBar` to simplify updating the progress bar when iterating.
```

Comment:
```suggestion
 * added an `iterate` method to `ProgressBar` to simplify updating the progress bar when iterating
```

Reply:
usually it's like `added ``ProgressBar::iterate()`` to ...`